### DISCUSSION TITLE
有关“每周一龙”期数计算的问题

### DISCUSSION THREAD

**Discussion by Antoneering** [loongson-community/areweloongyet#192] [General]
首先感谢您在于 LoongArch 社区资讯的介绍方面的辛勤工作。作为外行吃瓜人，您的资讯介绍的确令人受益匪浅。

但是，您最近的“每周一龙”栏目的标题期数计算似乎有些谬误。
以个人愚见，您在第 44 期与第 51 期由于断更，将命名改为 “每周两龙”，私以为，应该是“两周一龙”才对。毕竟您是将两周内容合为一篇，而不是一周内写了两篇。

再就您可能由于时间原因，可能存在无法及时更新而导致命名偏差的情况，或可采用“新近一龙”（有谐音“新进”之意），亦可简明称之“近期一龙”。

顺祝时祺！

**Comment by Antoneering** [comment]
抱歉，我好像放错地方了，应该是Issues里面。（手动狗头）

**Comment by xen0n** [comment]
> 首先感谢您在于 LoongArch 社区资讯的介绍方面的辛勤工作。作为外行吃瓜人，您的资讯介绍的确令人受益匪浅。
> 
> 但是，您最近的“每周一龙”栏目的标题期数计算似乎有些谬误。
> 以个人愚见，您在第 44 期与第 51 期由于断更，将命名改为 “每周两龙”，私以为，应该是“两周一龙”才对。毕竟您是将两周内容合为一篇，而不是一周内写了两篇。

这是故意的：此周报参考的原型之一 This Week in Rust 在类似情况下会改标题为 These Weeks in Rust -- 这也是每次“每周 N 龙”（N ≥ 2）时对应的英文 slug。为求形式上的一致性，由于汉语没有复数，只能改变唯一的数词。个人审美上，我会觉得“两周一龙”相比“每周两龙”，与“每周一龙”失去了更多对称；此外，如果理解成“每周”仍然有龙，但只是有的周被跳过了，所以下一周会覆盖两“龙”的内容 -- 此时“龙”是量词，也能说得过去。

总之，不打算改，希望您能理解 ;-)

> 再就您可能由于时间原因，可能存在无法及时更新而导致命名偏差的情况，或可采用“新近一龙”（有谐音“新进”之意），亦可简明称之“近期一龙”。
> 
> 顺祝时祺！

同上，由于希望保留英文名 This Week (These Weeks) in LoongArch，这个建议也无法直接采纳。。如果后续此周报体例会发生天翻地覆的改变，倒是可以考虑品牌更新了，但不是现在。

**Comment by xen0n** [comment]
Dupe of #193
